### ISSUE TITLE
Python server handler doesn't work on Ubuntu

### ISSUE THREAD

**Issue by tomezpl** [tomezpl/TuftePlaysPong#3]
Running server_handler.py (I suppose that's the right one) throws an exception (still renders, but doesn't seem to function at all):

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "server_handler.py", line 26, in messageHandler
    op=subprocess.Popen(["node","server.js"],stdin=subprocess.PIPE,stdout=subprocess.PIPE,stderr=subprocess.PIPE,creationflags=createFlags)
  File "/usr/lib/python2.7/subprocess.py", line 675, in __init__
    raise ValueError("creationflags is only supported on Windows "
ValueError: creationflags is only supported on Windows platforms
```

**Comment by KingCprey** [comment]
If statement checks if os.name=="nt" (Winders)
Otherwise should be k

**Comment by tomezpl** [comment]
Alrighty then, looks like it's fixed now (commit 35e8af3dd11635877d8f164ff89a679247cc2ae9)
